anyway/widgets/widget_utils.py

```diff
@@ -110,7 +112,7 @@ def run_query(query: db.session.query) -> Dict:
 
 
 def format_2_level_items(
-    items: Dict[str, dict],
+    items: Dict[typing.Union[int, str], dict],

```

Comment:
a dictionary with multiple key types might cause issues

Reply:
Yes, we have widgets with keys that are `int` and other as `str`, but the logic is the same, do you have an idea of how to solve it?

Reply:
well, standardize it  - either int or string

Reply:
Discussed on Slack